soc/xtensa/intel_adsp/common/soc_mp.c

```diff
@@ -66,6 +66,15 @@ static struct k_spinlock mplock;
 
 char *z_mp_stack_top;
 
+/* Vestigial silliness: An old mechanism for core startup would embed
+ * a "manifest" of code to copy to LP-SRAM at startup (vs. the tiny
+ * trampoline we use here).  This was constructed in the linker
+ * script, and the first word would encode the number of entries.  As
+ * it happens, SOF still emits the code to copy this data, so it needs
+ * to see this symbol point to a zero.
+ */
+uint32_t _loader_storage_manifest_start;
```

Comment:
Mostly because SOF internals are scarier and I wasn't sure what kind of thing that would break.  We don't want SOF to use the old-style IDC message code either, we just want it to do nothing.